lib/models/email/content.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'validates_type'
+
+module Models
+  module EMail
+    class Content
+      include ActiveModel::Model
+
+      attr_accessor :account, :amount, :charge_date, :value_date, :memo, :payee_id, :payee_name
+
+      validates_presence_of :account, :amount, :charge_date, :value_date, :memo
+      validates_numericality_of :amount
+      validates_type :charge_date, :date, allow_blank: false
+      validates_type :value_date, :date, allow_blank: false
```

Comment:
Doesn't

```suggestion
      validates_type :value_date, : charge_date, :date, allow_blank: false
```

work?

Reply:
might

Reply:
nope